### PR TITLE
Scrollbar with style added

### PR DESCRIPTION
**Pull request by zalabhavy**
Closes:  #103 

- **Title** : Change Style of Scrollbar
- **Name**:Zala Bhavy
- **Idenitfy yourself:** Gssoc'24

### Describe the add-ons or changes you've made 📃

I have changed in file index.css
Added Stylish Scrollbar instead of default scrollbar.
I added scrollbar with different blue shades.

### Type of change ☑️

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### Checklist: ☑️

- [x] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/Chanakya-Niti/blob/main/README.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/Chanakya-Niti/blob/main/CODE_OF_CONDUCT.md) of this project.
- [x] This PR does not contain plagiarized content.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] My changes generate no new warnings.

### Screenshots 📷
image

### CODE REVIEW

**Comment by zalabhavy** [comment]
@Avdhesh-Varshney Please check it and merge !!
Can you also assign me issue #106

File: src/index.css
Comment (Avdhesh-Varshney) [inline src/index.css:1]: @zalabhavy format the code by reducing the spacing of your code only.

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]

**Comment by zalabhavy** [comment]
@Avdhesh-Varshney Done please check it !

**Review by Avdhesh-Varshney — APPROVED** [review]
@zalabhavy PR Approved 🎉

**Pull request merged**